SU-SWS/stanford_basic
# D8CORE-3473 Fix the mobile menu search attributes

# READY FOR REVIEW

# Summary
- Adjusted the way that the mobile search form tweaks children attributes.

# Need Review By (Date)
- 3/3

# Urgency
- high.

# Steps to Test
1. checkout this branch
2. clear browser cache
3. reload any page
4. verify the mobile search field still displays and functions normally.
5. use the wave chrome tool to verify there are no errors with duplicate labels.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
